invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/FileUploader/FileUploader.js

```diff
@@ -130,7 +135,24 @@ export const FileUploaderComponent = ({
           </div>
         );
       } else {
-        uploadFiles(formikDraft, acceptedFiles);
+        if (!_isEmpty(emptyFiles) && !allowEmptyFiles) {
+          setWarningMsg(
+            <div className="content">
+              <Message
+                warning
+                icon="warning circle"
+                header="Could not upload files."
+                content={i18next.t("Empty files were skipped as they aren’t allowed.")}
```

Comment:
I would add translations for the header and remove the part `as they aren’t allowed`, as I think it is kind of normal?
```suggestion
                header=i18next.t("Could not upload all files.")
                content={i18next.t("Empty files were skipped.")}
```